spec/rack/test_spec.rb

```diff
@@ -356,6 +356,69 @@ def close
     end
   end
 
+  describe '#digest_authorize' do
```

Comment:
The unit tests for the digest auth has already been at `spec/rack/test/digest_auth_spec.rb`.
It might be good to add the new tests to `spec/rack/test/digest_auth_spec.rb`.
But in other words, in this time the code we want to test is at `lib/rack/test.rb`.
So, adding the unit tests at `spec/rack/test_spec.rb` might be good.

How do you think?